bike_data.py

```diff
@@ -0,0 +1,22 @@
+import requests
+import json
+
+#credentials
+api='https://api.tfl.gov.uk/bikepoint'
+application_id='0165cf3a'
+application_key='9187b8a0c05c716327bf753e55092b55'
+
+#requesting for the details of the bike
+resp=requests.get(api,auth=(application_id,application_key))
+
+#looping through the response object
+for each in resp.json():
+	#details of bikestation and its availability
+	bike_statn_name=each['commonName']
+	status=each['additionalProperties'][2]['key']
```

Comment:
The 2nd element might not always be the 'status' element, and this code will break when that assumption is invalidated. Can you think of an approach that would work regardless of which position this element is at in the same array?

Reply:
we can loop through the additional properties dict and searching could make it right.